frontend/src/tokens.css

```
@@ -0,0 +1,78 @@
+  --r3: 4px;
+
+  /* === SHADOWS === */
+  --shadow-sm: 0 1px 3px rgba(0, 0, 0, 0.4), 0 1px 2px rgba(0, 0, 0, 0.3);
+  --shadow-md: 0 4px 12px rgba(0, 0, 0, 0.5), 0 2px 6px rgba(0, 0, 0, 0.3);
+  --shadow-lg: 0 24px 80px rgba(0, 0, 0, 0.6), 0 8px 24px rgba(0, 0, 0, 0.4);
+  --shadow-err: 0 0 0 2px rgba(229, 83, 75, 0.3);
+
+  /* === TYPOGRAPHY & SPACING SCALE (Linear-like) === */
+  --font-sans: 'Inter', -apple-system, system-ui, sans-serif;
+  --font-display: 'Space Grotesk', -apple-system, system-ui, sans-serif;
+  --font-mono: ui-monospace, SFMono-Regular, Menlo, Monaco, Consolas, 'Liberation Mono', 'Courier New', monospace;
```

Stylelint (17.5.0)

[error] Expected "SFMono-Regular" to be "sfmono-regular" (value-keyword-case)

(value-keyword-case)